Update 00-sql-introduction.md (#353)
datacarpentry/sql-ecology-lesson: episodes/00-sql-introduction.md

### Definition: *Relational Database*

> A relational database stores data in *tables* made up of *records* (rows) and *attributes* (columns).

Reviewer: Omit this line.

> *Attributes* may also be called *fields*, as *records* may also be called *observations*; Depending

Reviewer: Please omit.

> on the scientific topics and what is being recorded in the database, the jargons change a little bit.

Reviewer: Please omit this.

> 

Reviewer: ```suggestion
In most cases, each record will have a unique identifier, called a *key*, which is stored as one of its fields.
```

> The idea of *relational* is associated with organizing information across multiple tables in the database.

Reviewer: Please keep the remaining lines from here on.

> The *relationship* between tables in the database can be established (or even enforced) in different ways,

Reviewer: ```suggestion
The *relationships* between tables in a database can be established (or even enforced) in different ways,
```

> from a pure conceptual decision to the use of *keys* and other mechanisms.

Reviewer: ```suggestion
through the use of *keys* and other constraints.
```

In many small datasets we have only a few tables and we don't have to think about cross-references

> within data sets. 

Reviewer: ```suggestion
within datasets. 
```

The idea of *relational data* becomes more and more important as the amount and complexity of data grows.

> And that is really the motivation for relational databases: to organize data through related sets.

Reviewer: ```suggestion
This provides the motivation for using relational databases: to organize data through related keys.
```

> And that is really the motivation for *relational databases*: to organize data in small, topic-related sets.

Reviewer: ```suggestion
And that is really the motivation for relational databases: to organize data in through related sets.
```

> Records in one tables may contain keys that refer to records in other tables,

Reviewer: ```suggestion
Records in one table may contain keys that refer to records in other tables,
```
